hoodie-spark/src/test/scala/DataSourceTest.scala

```diff
@@ -131,6 +131,7 @@ class DataSourceTest extends AssertionsForJUnit {
     val inputDF1: Dataset[Row] = spark.read.json(spark.sparkContext.parallelize(records1, 2))
     inputDF1.write.format("com.uber.hoodie")
       .options(commonOpts)
+      .option("hoodie.compact.inline", "false") // else fails due to compaction & deltacommit instant times being same
```

Comment:
Did not follow the comments you made in this line...

Reply:
since test runs very quickly, the delta commit and compaction start in the same second. Don't know how to prevent this per se.

Reply:
I see, put a sleep(1s) for now ?